tests/gitgitgadget.test.ts

```diff
@@ -25,8 +25,8 @@ MIME-Version: 1.0
 To: reviewer@example.com
 Cc: Some Body <somebody@example.com>
 
-This Pull Request contains some really important changes that I would love to
-have included in git.git.
+This Pull Request contains some really important changes that I would love
+to have included in git.git [https://github.com/git/git].
```

Comment:
Did you mean `[git.git](https://github.com/git/git)` ?

Reply:
No, this is the output of the renderer. The renderer specifically does not use Markdown to show links, but shows link texts as plain text, followed by the link in brackets.

It is the default of the [`html-to-text`](https://www.npmjs.com/package/html-to-text) package that I use to do all the hard work.